src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Dsa.cs

```diff
@@ -65,7 +65,7 @@ internal static int DsaKeySize(SafeDsaHandle dsa)
             // there were some leading zero bytes truncated due to BIGNUM semantics.
             // This minimizes the change the key size is wrong due to leading zero bits; there is only
             // a 2^64 chance that the key has 64 leading zero bits and thus the keysize will be wrong.
```

Comment:
Also, that formula is in bytes, not bits. So it will round up to nearest 8 byte boundary.